src/main/java/core/basesyntax/DataConverterImpl.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverterMet {
+    private final String comma = ",";

```

Comment:
```suggestion
    private static final String COMMA = ",";
```
use the correct format of constant